argo-workflows/okd-workflows/20-image-stream.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: image.openshift.io/v1
+kind: ImageStream
+metadata:
+  name: release
+spec:
+  lookupPolicy:
```

Comment:
shouldn't the importMode be set to "PreserveOriginal" as well?

Reply:
everyhing should be set